### PR TITLE
Fixes #685

### LINKED ISSUE #685: Space/Ellipsis text  messed up in HIG
See https://elementary.io/docs/human-interface-guidelines#prevent-common-mistakes, compare to http://old.elementary.io/docs/human-interface-guidelines/punctuation.

> * Only use **one space** after a period.
>   * i.e. “This is a sentence. This is another sentence.”
>   * Use `\u0133` in your code.
>   * See [Using Ellipses](#) for usage details.

Should be:

> * Only use **one space** after a period.
>   * i.e. “This is a sentence. This is another sentence.”
> * Use a proper **ellipsis character** (…) rather than three dots.
>   * Use `\u0133` in your code.
>   * See [Using Ellipses](#) for usage details.

### PR DESCRIPTION
**Pull request by aleksandar-todorovic**

### CODE REVIEW

File: docs/human-interface-guidelines.md [hunk 1]
```diff
@@ -902,8 +902,9 @@ Proper typography is important throughout elementary OS. Not just for consistenc
 
 * Only use **one space** after a period.
     * i.e. “This is a sentence. This is another sentence.”
-    * Use `\u0133` in your code.
-    * See [Using Ellipses](#using-ellipses) for usage details.
+* Use a proper **ellipsis character** (…) rather than three dots.
+    Use `u0133` in your code.
+    See [Using Ellipses](https://elementary.io/docs/human-interface-guidelines#using-ellipses) for usage details.
```
Comment (cassidyjames) [inline docs/human-interface-guidelines.md]: The existing link was correct. Revert that and this looks good.

File: docs/human-interface-guidelines.md [hunk 2]
```diff
@@ -902,8 +902,9 @@ Proper typography is important throughout elementary OS. Not just for consistenc
 
 * Only use **one space** after a period.
     * i.e. “This is a sentence. This is another sentence.”
-    * Use `\u0133` in your code.
-    * See [Using Ellipses](#using-ellipses) for usage details.
+* Use a proper **ellipsis character** (…) rather than three dots.
+    Use `u0133` in your code.
```
Comment (cassidyjames) [inline docs/human-interface-guidelines.md]: Oh, should be `\u0133` like before, including the backslash

**Comment by lewisgoddard** [comment]
1. The link doesn't need to be changed to absolute, it works already.
2. You seem to be missing the `*` from the beginning of the two indented lines now.
3. The leading `\` before the character code is gone too.

**Comment by cassidyjames** [comment]
@lewisgoddard ha yeah good review, I missed the missing `*`s.

**Comment by lewisgoddard** [comment]
Closing in favor of #703, thank you for your contribution.

**Pull request closed without merging**